tests/integration/integration_test.go

```diff
@@ -345,44 +346,44 @@ func genOperatorConfig(t *testing.T, ctx context.Context, mainnetAnvil *AnvilIns
 	os.Setenv("OPERATOR_BLS_KEY_PASSWORD", "")
 	os.Setenv("OPERATOR_ECDSA_KEY_PASSWORD", "")
 
-	err = os.MkdirAll(getOperatorKeysPathPrefix(t), 0770)
+	nodeConfig.BlsPrivateKeyStorePath, err = filepath.Abs(filepath.Join("../keys/bls", keysPath, "key.json"))
 	if err != nil {
-		t.Fatalf("Failed to create operators keys dir: %s", err.Error())
+		t.Fatalf("Failed to get BLS key dir: %s", err.Error())
 	}
-
-	keysPath, err := os.MkdirTemp(getOperatorKeysPathPrefix(t), "")
+	passwordPath := filepath.Join("../keys/bls", keysPath, "password.txt")
```

Comment:
Let's declare const it the scope of the function.
```suggestion
	passwordPath := filepath.Join(BlsKeysDir, keysPath, "password.txt")
```

Reply:
Done, let it in the test package scope, as the other dir - [450c845](https://github.com/NethermindEth/near-sffl/pull/71/commits/450c845b4021af70c0480f82ae87e46af23bb686)